assets/grafana/deployment.yaml

```
@@ -137,6 +137,36 @@ spec:
         - mountPath: /etc/proxy/secrets
           name: secret-grafana-proxy
           readOnly: false
+      - args:
+        - --secure-listen-address=0.0.0.0:9097
```

(nit) let continue in the same port range

```suggestion
        - --secure-listen-address=0.0.0.0:3002
```